src/main/java/com/twentyonec/ItemsLogger/commands/MainCommand.java

```diff
@@ -0,0 +1,30 @@
+package com.twentyonec.ItemsLogger.commands;
+
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandExecutor;
+import org.bukkit.command.CommandSender;
+
+import com.twentyonec.ItemsLogger.utils.Permissions;
+
+public class MainCommand implements CommandExecutor{
+
+	@Override
+	public boolean onCommand(final CommandSender sender, final Command command, final String label, final String[] args) {
+
+		CommandInterface cmd = null;
```

Comment:
You don't have to initialize this to `null`. Declaration without assignment is absolutely allowed
```suggestion
		CommandInterface cmd;
```